auto_round/export/export_to_itrex/export.py

```diff
@@ -127,6 +180,9 @@ def pack_model(
         if not isinstance(scale, torch.Tensor):
             scale = torch.tensor(scale, dtype=convert_dtype)
             zp = torch.tensor(zp, dtype=torch.int32)
+            if device == "xpu":
+                scale = torch.tensor(v["scale"], dtype=torch.float32)
```

Comment:
better add a log here, if convert_dtype != torch.float32 there maybe lead to loss of acc